css/styles.css

```diff
@@ -99,4 +141,33 @@
 .next-button-wrapper {
   margin-right: 15px;
   padding: 14px 0;
+  padding-left: 5px;
+}
+
+.study-button-wrapper {
+  padding: 10px 15px;
+}
+
+.study-button {
+  font-family: Roboto, sans-serif;
+  width: 65%;
+  height: 65px;
+  border: none;
+  border-radius: 0.4em;
+  background-color: #015c75;
+  color: white;
+  font-size: 45px;
+  position: fixed;
+  bottom: 1%;
+  left: 17%;
```

Comment:
Shouldn't this be `17.5%`? I would think this would make your button slightly off-center